front/components/assistant/AssistantDetailsDropdownMenu.tsx

```diff
@@ -67,7 +70,7 @@ export function AssistantDetailsDropdownMenu({
   const isGlobalAgent = agentConfiguration.scope === "global";
 
   const isInList = agentConfiguration.userListStatus === "in-list";
-  const canDelete = onAgentDeletion && (isBuilder(owner) || !isAgentWorkspace);
+  const allowDeletion = canDelete && (isBuilder(owner) || !isAgentWorkspace);
```

Comment:
`nit`: Could we group these:
```
 if (!agentConfiguration || !user) {
    return <></>;
  }
  if (agentConfiguration.status === "archived") {
    return <></>;
  }
  ```
  into one if statement?

Reply:
Yep !